modules/avoindata-drupal-theme/config/install/language/sv/easy_breadcrumb.settings.yml

```diff
@@ -0,0 +1,2 @@
+home_segment_title: Frontsidan
```

Comment:
Framsidan?

Reply:
Changed this for the main header link as well. At least according to sanakirja.org, it would be "hemsida" so I changed it to that.